src/components/Questionnaire/Questions/Questions.vue

```diff
@@ -67,8 +84,20 @@ export default defineComponent({
       required: true
     }
   },
-  setup() {
-    return {}
+  setup(props) {
+    const createRange = (max: number, min: number) => {
+      const range = []
+      for (let i = min; i <= max; i++) {
+        range.push(i)
+      }
+      return range
+    }
+    const questionrange = props.questioncontents.reduce((a, q) => {
+      if (q.question_type === 'LinearScale')
+        a[q.questionID] = createRange(q.scale_max, q.scale_min)
+      return a
+    }, {} as questionrangedetail)
```

Comment:
```suggestion
    const questionrange = props.questioncontents.reduce((acc, q) => {
      if (q.question_type === 'LinearScale')
        acc[q.questionID] = createRange(q.scale_max, q.scale_min)
      return acc
    }, {} as questionrangedetail)
```
`a`じゃわかんない(最初適当につけた変数名だと思った)から`acc`にしてほしい